### DISCUSSION TITLE
Mengirim data berukuran besar melalui API

### DISCUSSION THREAD

**Discussion by rezhajamil** [deaafrizal/forum#37] [Troubleshooting Room]
Izin diskusi agan cuy semua

Kasusnya ini ada 2 sistem / website, masing-masing pakai PHP (CI 3 & Laravel) pada server yang berbeda (Internal Company Server & Hosting VPS dari luar).
Nah, **GOALS** nya adalah melakukan pengiriman data dari database MYSQL pada server internal ke server luar melalui API setiap harinya.
Sudah saya buat dengan menjalankan backendnya di PHP dan kirim datanya melalui AJAX nya JQuery. Dan sudah berhasil (testing puluhan data), datanya sudah masuk ke database di server 1 nya lagi.

Nah yang jadi **PROBLEM** adalah data yang dikirim jumlahnya puluhan ribu s/d ratusan ribu row, sehingga terjadi ERROR melebih limit memory, size, waktu.

Nah dari agan cuy sekalian ada gak kira-kira saran / experience dengan kasus mirip?
Ada gak ya solusi agar bisa datanya dikirim tanpa harus nge modif settingan limit di server nya?

Pun10

**Comment by fandyaditya** [comment]
Apakah mesti setiap hari dalam 1 waktu kirim semua sekaligus? Kalau tidak, bisa pake queue, cicil kirim per batch
